Sources/Confidence/ConfidenceValue.swift

```diff
@@ -220,6 +220,7 @@ extension ConfidenceValueInternal {
             }
         case .timestamp(let date):
             let isoFormatter = ISO8601DateFormatter()
+            isoFormatter.timeZone = TimeZone.current
```

Comment:
The default is GMT: this ensures that the local TimeZone of the device is transmitted to the backend (still under the ISO8601 standards)